jupyterlab_git/git.py

```diff
@@ -1681,6 +1681,22 @@ async def remote_remove(self, path, name):
 
         return response
 
+    async def read_file(self, path):
+        """
+        Reads file content located at path and returns it as a string
+
+        path: str
+            The path of the file
+        """
+        try:
+            file = pathlib.Path(path)
+            if file.stat().st_size > 0:
```

Comment:
I just wanted to check if the file was empty and then return an empty string if it was. I guess since if the file is empty, then it would be an empty string regardless. I will remove it.